serde-support/src/test/groovy/io/micronaut/serde/support/serdes/BeanContextSerdeSpec.groovy

```diff
@@ -71,5 +73,21 @@ class BeanContextSerdeSpec extends Specification {
         enumSerde.getClass() == EnumSerde
     }
 
+    void "test retrieving Map deserializer"() {
+        when:
+        def deserializer = beanContext.getBean(Argument.of(Deserializer, Argument.mapOf(String, Object)))
+
+        then:
+        deserializer
+    }
+
+    void "test retrieving Map serializer"() {
+        when:
+        def deserializer = beanContext.getBean(Argument.of(Serializer, Argument.mapOf(String, Object)))
+
+        then:
+        deserializer
```

Comment:
add tests that actually use the serialiser/deserializer to serialize

Reply:
Added